react-app/src/components/Form/content/PackageSection.tsx

```diff
@@ -13,7 +17,11 @@ export const getIdLabel = (authority: AuthorityUnion) => {
   return idLabels.get(authority) ?? BLANK_VALUE;
 };
 
-export const getAuthorityLabel = (authority: AuthorityUnion) => {
+export const getAuthorityLabel = (authority: AuthorityUnion | null | undefined) => {
```

Comment:
Will `authority` will ever be `null` if it's coming from params? If yes, then it's possible for `authority` to be anything else, and there's no value in making this change

Reply:
Typescript complains